modules/client/test/integration/client/decoding.test.ts

```diff
@@ -504,6 +505,77 @@ describe("Client", () => {
         bytesToHex(decodedUpgradeToAndCallParams.data),
       );
     });
+    it("Should decode an apply uninstallation action", async () => {
+      const networkSpy = jest.spyOn(JsonRpcProvider.prototype, "network", "get");
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const applyUninstallationParams: ApplyUninstallationParams = {
+        permissions: [{
+          condition: "0x1234567890123456789012345678901234567890",
```

Comment:
Why do we need a condition?

Reply:
It is required by the ethers-contracts parameters, they expect a `MultiTargetPermissionStruct` defined as:

```ts
 type MultiTargetPermissionStruct = {
        operation: BigNumberish;
        where: string;
        who: string;
        condition: string;
        permissionId: BytesLike;
    };
```